app/client/mindjogg/.eslintrc.json

```diff
@@ -21,5 +21,10 @@
         "react"
     ],
     "rules": {
+        "prefer-const": "warn",
+        "quotes": "warn",
+        "no-inline-comments": "warn",
+        "semi-style": "warn",
```

Comment:
@ja-chisholm can you please change to "error" to every rules except "prefer-const" because we the developer have the tendency to overlook the warnings.

Reply:
Just updated